apps/logging/signals.py

```diff
@@ -40,9 +37,7 @@
 @receiver(app_authorized)
 def handle_token_created(sender, request, token, **kwargs):
     # Get auth flow dict from session for logging
-    auth_flow_dict = get_session_auth_flow_trace(request)
-
-    token_logger.info(get_event(Token(token, action="authorized", auth_flow_dict=auth_flow_dict)))
+    token_logger.info(Token(token, action="authorized", auth_flow_dict=get_session_auth_flow_trace(request)))
```

Comment:
```suggestion
    request._logger.info(Token(token, action="authorized", auth_flow_dict=get_session_auth_flow_trace(request)))
```

Reply:
I think we had also talked about just logging dicts, so we might want to transform this into a dict instead of using the Token... what did you think @JFU-GIT , @dtisza1

Reply:
I tried to keep the code at point of logging un-changed because there are many places, don't want to intro error by re-factor many places, instead, logic added in RequestLogger.format(...), to invoke a method to_dict where a dict is extracted from the object, it pretty much equivalent to convert to dict at the calling place.

Reply:
I would probably keep the value being sent to `info` as a built in data value still. Otherwise it just gets confusing because we are adding conversion functions to other objects without really knowing the purpose. Like if brand new people were to start writing code for BB2, they would see this and think logging could take any object. Then start passing objects and not get expected results.

Basically just convert the object to a dict in line:

```
request._logger.info(Token(token, action="authorized", auth_flow_dict=get_session_auth_flow_trace(request)).to_dict())
```

Reply:
> I would probably keep the value being sent to `info` as a built in data value still. Otherwise it just gets confusing because we are adding conversion functions to other objects without really knowing the purpose. Like if brand new people were to start writing code for BB2, they would see this and think logging could take any class. Then start passing objects and not get expected results.
> 
> Basically just convert the object to a dict in line:
> 
> ```
> request._logger.info(Token(token, action="authorized", auth_flow_dict=get_session_auth_flow_trace(request)).to_dict())
> ```

Sure can refactor calling places and convert to dict there.
the original thinking is to be less intrusive hence reduce the chance of introducing changes to the logging events from different loggers...